agents/postgres/pgstatmonitor/pgstatmonitor.go

```diff
@@ -239,7 +239,6 @@ func (m *PGStatMonitorQAN) makeBuckets(current, cache map[time.Time]map[string]*
 					IsTruncated:         currentPSM.IsQueryTruncated,
 					Fingerprint:         currentPSM.Fingerprint,
 					Database:            currentPSM.Database,
-					Tables:              currentPSM.TablesNames,
```

Comment:
TablesNames was renamed to Relations, so please rename it instead of removing this field.

Reply:
https://github.com/percona/pg_stat_monitor/blob/master/docs/USER_GUIDE.md

Reply:
Oh, thank you! I looked for it but not found.

Reply:
Renamed.

Reply:
Should we rename `MetricsBucket_Common` field?